sdk/src/main/scala/io/horizen/utxo/SidechainApp.scala

```diff
@@ -180,55 +183,64 @@ class SidechainApp @Inject()
     sidechainWalletCswDataProvider,
     backupStorage,
     params,
+    sc2scConfigurator,
     timeProvider,
     applicationWallet,
     applicationState,
     genesisBlock
-    ) // TO DO: why not to put genesisBlock as a part of params? REVIEW Params structure
+  ) // TO DO: why not to put genesisBlock as a part of params? REVIEW Params structure
 
   def modifierSerializers: Map[ModifierTypeId, SparkzSerializer[_ <: NodeViewModifier]] =
     Map(SidechainBlockBase.ModifierTypeId -> new SidechainBlockSerializer(sidechainTransactionsCompanion),
       Transaction.ModifierTypeId -> sidechainTransactionsCompanion)
 
   override val nodeViewSynchronizer: ActorRef =
     actorSystem.actorOf(SidechainNodeViewSynchronizer.props(networkControllerRef, nodeViewHolderRef,
-        SidechainSyncInfoMessageSpec, settings.network, timeProvider, modifierSerializers))
+      SidechainSyncInfoMessageSpec, settings.network, timeProvider, modifierSerializers))
 
   // Init Forger with a proper web socket client
-  val sidechainBlockForgerActorRef: ActorRef = forge.ForgerRef("Forger", sidechainSettings, nodeViewHolderRef,  mainchainSynchronizer, sidechainTransactionsCompanion, timeProvider, params)
+  val sidechainBlockForgerActorRef: ActorRef = ForgerRef("Forger", sidechainSettings, nodeViewHolderRef, mainchainSynchronizer, sidechainTransactionsCompanion, timeProvider, sc2scConfigurator, params)
```

Comment:
A side comment to NetworkParams class.
You may see in this file a lot of places where we pass `sc2scConfigurator` to a constructor altogether with `params`. Can be simplified a lot by putting it inside `params` once.